packages/core/package.json

```diff
@@ -21,5 +23,8 @@
     "@types/lodash": "^4.14.170",
     "@types/webpack-env": "^1.16.2",
     "tslib": "^2.3.0"
+  },
+  "peerDependencies": {
+    "aws-amplify": "3.x.x || 4.x.x"
```

Comment:
Otherwise our `amazon-cognito-identity-js` and `aws-amplify` dependencies get bundled!